src/lib/wasm/MultipassStore.ts

```diff
@@ -32,7 +32,9 @@ class MultipassStore {
         if (multipass) {
             try {
                 await multipass.create_identity(username, passphrase);
-                await this.updateStatusMessage(statusMessage);
+                if (statusMessage.length > 0) {
```

Comment:
might want to also check to make sure it doesnt exceed a specific length otherwise it may still give an error.

Reply:
done